scriptmodules/emulators/reicast.sh

```diff
@@ -42,7 +42,15 @@ function configure_reicast() {
 
     # create bios dir. Copy dc_boot.bin and dc_flash.bin there.
     mkdir $md_inst/data
+
+    cat > $md_inst/reicast.sh << _EOF_
+#!/bin/bash
+cd /opt/retropie/emulators/reicast
+sudo ./reicast.elf -config config:image="$1"
```

Comment:
you can quote it

```
\$1
```

instead of using cd btw you could use pushd / popd so if someone launches the .sh they stay in their current folder on exit. also you can use $md_inst instead of the /opt/retropie/emulators/reicast path

if you don't need any variables to be used in a script you are writing out using 

```
<< _EOF_
```

you can add a backslash

```
<< \_EOF
```

which we use for example in bashwelcometweak